tests/test_relion_interop.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+import numpy as np
+import pytest
+
+from aspire.source import RelionSource, Simulation
+from aspire.volume import Volume
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
+
+
+STARFILE = ["rln_proj_65.star", "rln_proj_64.star"]
+
+
+@pytest.fixture(params=STARFILE, ids=lambda x: f"resolution={x}", scope="module")
+def starfile(request):
```

Comment:
this starfile fixture is not necessary anyway... can rm.

Reply:
removed.